### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by swaptr**
Added  install, code style, built with, support and license sections in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
 # kotlin-template
-A minimal template for projects base on JRPCLightning. This should get you started if you want develop a c-lightning plugin with java env
+A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a c-lightning plugin using Java.
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a core lightning plugin using Java.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
...
+
+## Install
+Java produces a jar and c-lightning needs a bash script to run it!
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
Java produces a jar and core lightning needs a bash script to run it!
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,63 @@
...
+The gradle script compiles the plugin and generate a bash script using:
+```bash
+./gradlew formatKotlin
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
./gradlew createRunnableScript
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
 # kotlin-template
-A minimal template for projects base on JRPCLightning. This should get you started if you want develop a c-lightning plugin with java env
+A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a c-lightning plugin using Java.
+
+## Install
+Java produces a jar and c-lightning needs a bash script to run it!
+The gradle script compiles the plugin and generate a bash script using:
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
The gradle is able to compile the plugin in a jar, and generate a runnable script that core lightning use to run it
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,63 @@
...
+
+
+## Built with
+
+- [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning)
+
+## Support
+If you like the library and want to support it, please considerer to donate with the following system
+
+
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,63 @@
...
+- [3BQ8qbn8hLdmBKEjt1Hj1Z6SiDsnjJurfU](bitcoin:3BQ8qbn8hLdmBKEjt1Hj1Z6SiDsnjJurfU)
+- [Github support](https://github.com/sponsors/vincenzopalazzo)
+
+## License
+
+<div align="center">
+  <img src="https://opensource.org/files/osi_keyhole_300X300_90ppi_0.png" width="150" height="150"/>
+</div>
+
+It is a c-lightning plugin to override Bitcoin backend plugin with esplora.
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
It is a core lightning plugin to override Bitcoin backend plugin with esplora.
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,63 @@
...
+
+Copyright (C) 2020-2021 Vincenzo Palazzo vincenzopalazzodev@gmail.com
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
Copyright (C) 2020-2022 Vincenzo Palazzo vincenzopalazzodev@gmail.com
``

File: README.md (hunk 6 above)
Comment (vincenzopalazzo) [reply README.md]: Maybe it is better

>It is a core lightning plugin template to develop plugin with Kotlin JVM

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Very good Job, thanks!

Do you think these are good addition?

**Review by swaptr — COMMENTED** [review]
Applied changes and open for review.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,63 @@
 # kotlin-template
-A minimal template for projects base on JRPCLightning. This should get you started if you want develop a c-lightning plugin with java env
+A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a core lightning plugin using Java.
```
Comment (vincenzopalazzo) [inline README.md]:
```suggestion
A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a core lightning plugin using Kotlin.
```

Sorry I missed it before

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM, can you squash the commit in a single one?

`git rebase -i HEAD~6` should open an editor with the list of commits, you can squash all, and keep only the first one.

In the end when you make the squash you need a `git push -f`

**Review by vincenzopalazzo — APPROVED** [review]
Amazing!

Thanks!

**Pull request merged**